Documentation/policy/caveats.rst

```diff
@@ -25,3 +25,22 @@ selected endpoint by the LB. If not, i.e., the request needs to be forwarded to
 another node after the service endpoint selection, then it will have the ``reserved:remote-node``.
 
 The latter traffic will match ``fromEntities: cluster`` policies.
+
+Differences From Kubernetes Network Policies
+============================================
+
+When creating Cilium Network Policies it is important to keep in mind that Cilium Network
+Policies do not perfectly replicate the functionality of `Kubernetes Network Policies <https://kubernetes.io/docs/concepts/services-networking/network-policies/>`_.
+
+There are two ways Cilium Network Policies do not overlap with existing Kubernetes Network
+Policy functionality:
+
+1. Cilium Network Policies that reference the Stream Control Transmission Protocol (SCTP)
+   will not work properly. Currently, Cilium does not support SCTP.
+
+2. Cilium Network Policies that use CIDR blocks to define endpoints controlled by Cilium
+   (i.e. internal to the Kubernetes cluster) will not work properly. As stated elsewhere
+   in this documentation, CIDR policies in Cilium are used to define policies to and from
+   endpoints which are not managed by Cilium (i.e. external to the Kubernetes cluster).
+   CIDR policies **can** be used in Kubernetes network policies to define policies to and
+   from endpoints which are internal to the Kubernetes cluster (i.e. managed by the CNI).
```

Comment:
I think this is a little bit ambiguous, as it could imply that Cilium implements upstream Kubernetes network policies in a way that allows those CIDR statements to apply to cilium-managed endpoints. However, that is not the case.